.github/workflows/e2e-regressions.yml

```diff
@@ -0,0 +1,61 @@
+name: e2e-regression
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
I feel like there's a lot of overhead here for nightly running (e.g. 5 minutes to build the docker images, to run a test for a minute)

Reply:
I suppose, but I'm not clear on why it matters? They're run nightly without us having to really do anything and we don't wait on them to complete.